app/lib/crypto/hash.ts

```diff
@@ -18,3 +18,22 @@ const hmacFunction = (hashFunctionName: string, key: Buffer) => {
 export const sha512hmac = (key: Buffer, data: Buffer) => {
   return hmacFunction("sha512", key)(data)
 }
+
+/**
+ * Create sha function for the indicated hash function
```

Comment:
More expressive comment:
`Wrap crypto.createHash to streamline its worflow without losing parameterization.`